src/libraries/System.Security.Cryptography.Cose/src/System/Security/Cryptography/Cose/CoseMultiSignMessage.cs

```diff
@@ -379,9 +670,41 @@ public override bool TryEncode(Span<byte> destination, out int bytesWritten)
             return true;
         }
 
+        /// <summary>
+        /// Adds a signature for the content embedded in this message.
+        /// </summary>
+        /// <param name="signer">The signer information used to sign the content.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must also be provided during verification.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="signer"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///     The <see cref="CoseSigner.ProtectedHeaders"/> and <see cref="CoseSigner.UnprotectedHeaders"/> collections in <paramref name="signer"/> have one or more labels in common.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     One of the labels specified in a <see cref="CoseHeaderLabel.CriticalHeaders"/> header is missing.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="InvalidOperationException">Content is detached from this message, use an overload that accepts a detached content.</exception>
         public void AddSignatureForEmbedded(CoseSigner signer, byte[]? associatedData = null)
             => AddSignatureForEmbedded(signer, associatedData.AsSpan());
 
+        /// <summary>
+        /// Adds a signature for the content embedded in this message.
+        /// </summary>
+        /// <param name="signer">The signer information used to sign the content.</param>
+        /// <param name="associatedData">The extra data associated with the signature, which must also be provided during verification.</param>
+        /// <exception cref="ArgumentNullException"><paramref name="signer"/> is <see langword="null"/>.</exception>
+        /// <exception cref="ArgumentException">
+        ///   <para>
+        ///     The <see cref="CoseSigner.ProtectedHeaders"/> and <see cref="CoseSigner.UnprotectedHeaders"/> collections in <paramref name="signer"/> have one or more labels in common.
+        ///   </para>
+        ///   <para>-or-</para>
+        ///   <para>
+        ///     One of the labels specified in a <see cref="CoseHeaderLabel.CriticalHeaders"/> header is missing.
+        ///   </para>
+        /// </exception>
+        /// <exception cref="InvalidOperationException">Content is detached from this message, use an overload that accepts a detached content.</exception>
```

Comment:
Since we're talking about the concept of content vs the Content property, we should probably have a "the" here.

Think of it as when I say things like "Me gusta cafe." when I clearly mean "Me gusta el cafe."  Except, of course, entirely different :wink:.

```suggestion
        /// <exception cref="InvalidOperationException">The content is detached from this message, use an overload that accepts a detached content.</exception>
```